### PR TITLE
UCP/WIREUP/ATOMICS: use 2 bits (32/64) to request atomics

### PR DESCRIPTION
**Pull request by hoopoepg**
- added 2 flags to request atomics on peer
- updated iface query to set aggregate atomics ops

### CODE REVIEW

File: src/uct/api/uct.h [hunk 1]
```diff
@@ -204,6 +204,9 @@ typedef enum uct_atomic_op {
 #define UCT_IFACE_FLAG_ATOMIC_CSWAP32 UCS_BIT(22) /**< 32bit atomic compare-and-swap */
 #define UCT_IFACE_FLAG_ATOMIC_CSWAP64 UCS_BIT(23) /**< 64bit atomic compare-and-swap */
 
+#define UCT_IFACE_FLAG_ATOMIC32       UCS_BIT(24) /**< 32bit atomic operations */
```
Comment (yosefe) [inline src/uct/api/uct.h]: @hoopoepg i think we should change it only in UCP address packing functions, not in UCT API

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6730/ for details (Mellanox internal link).

File: src/uct/api/uct.h (hunk 1 above)
Comment (hoopoepg) [reply src/uct/api/uct.h]: fixed

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6731/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6732/ for details (Mellanox internal link).

**Pull request merged**